README.md

```diff
@@ -12,7 +12,7 @@
 
 # Introduction
 
-The Bluetooth Profile Tuning Suite (PTS) is a Bluetooth testing tool provided by Bluetooth SIG. The PTS is a Windows program that is normally used in manual mode via its GUI
+The Bluetooth Profile Tuning Suite (PTS) is a Bluetooth testing tool provided by Bluetooth SIG. The PTS is a Windows program that is normally used in manual mode via its GUI.
```

Comment:
Are you going to extend this PR somehow e.g. by fixing more coding style issues?

Reply:
nope

Reply:
thank you for your contribution